csharp/DateTimeNanos.cs

```diff
@@ -10,14 +10,37 @@ namespace ParquetSharp
     [StructLayout(LayoutKind.Sequential)]
     public readonly struct DateTimeNanos : IEquatable<DateTimeNanos>, IComparable, IComparable<DateTimeNanos>
     {
+        private const long DateTimeOffsetTicks = 621355968000000000; // new DateTime(1970, 01, 01).Ticks
+        private const string DefaultFormat = "yyyy-MM-dd HH:mm:ss.fffffffff";
```

Comment:
Can you please leave private members at the bottom of the class definition for consistency with the ParquetSharp codebase.